frontend/src/metabase-lib/types.ts

```diff
@@ -198,6 +198,7 @@ export type ExpressionOperatorName =
   | "not-empty"
   | "starts-with"
   | "ends-with"
+  | "concat"
```

Comment:
For testing only. I need a valid string expression that is not handled by our string filter UI.